openbb_terminal/core/models/fmp.py

```diff
@@ -0,0 +1,840 @@
+        """Get historical income statement growth
+
+        Parameters
+        ----------
+        symbol : str
+            Symbol to get data for
+        limit : int, optional
+            Number of statements to get, by default 200
+
+        Returns
+        -------
+        pd.DataFrame
+            Historical income statement growth
+
+        Examples
+        --------
+        >>> from openbb_terminal.sdk import openbb
+        >>> income_growth = openbb.fmp.income_growth("AAPL")
+        """
+        url = f"{cls._BASE_URL()}income-statement-growth/{symbol}?"
+        r = request(url + f"apikey={cls._KEY()}&limit={limit}")
+        return pd.DataFrame(cls.validate_request(r)).T
+
+    @classmethod
+    def balance_growth(cls, symbol: str, limit: int = 200) -> pd.DataFrame:
+        """Get historical balance sheet growth
+
+        Parameters
+        ----------
+        symbol : str
+            Symbol to get data for
+        limit : int, optional
+            Number of statements to get, by default 200
+
+        Returns
+        -------
+        pd.DataFrame
+            Dataframe of historical balance sheet growth
+
+        Examples
+        --------
+        >>> from openbb_terminal.sdk import openbb
+        >>> balance_growth = openbb.fmp.balance_growth("AAPL")
+        """
+        url = f"{cls._BASE_URL()}balance-sheet-statement-growth/{symbol}?"
+        r = request(url + f"apikey={cls._KEY()}&limit={limit}")
+        return pd.DataFrame(cls.validate_request(r)).T
+
+    @classmethod
+    def cash_flow_growth(cls, symbol: str, limit: int = 00) -> pd.DataFrame:
+        """Get historical cash flow growth
+
+        Parameters
+        ----------
+        symbol : str
+            Symbol to get data for
+        limit : int, optional
+            Number of statements to get, by default 200
+
+        Returns
+        -------
+        pd.DataFrame
+            Dataframe of historical cash flow growth
+
+        Examples
+        --------
+        >>> from openbb_terminal.sdk import openbb
+        >>> cash_growth = openbb.fmp.cash_growth("AAPL")
+        """
+        url = f"{cls._BASE_URL()}cash-flow-statement-growth/{symbol}?"
+        r = request(url + f"apikey={cls._KEY()}&limit={limit}")
+        return pd.DataFrame(cls.validate_request(r)).T
+
+    @classmethod
+    def revseg(cls, symbol: str, quarter=False) -> dict:
+        """Get sales revenue by segment
+
+        Parameters
+        ----------
+        symbol : str
+            Symbol to get data for
+        quarter : bool, optional
+            Flag to get quarterly data, by default False
+
+        Returns
+        -------
+        dict
+            Dictionary containing the revenue by segment
+
+        Examples
+        --------
+        >>> from openbb_terminal.sdk import openbb
+        >>> revseg = openbb.fmp.revseg("AAPL")
+        """
+        # TODO: make dataframe since this is not a fixed shape
```

Comment:
was about to ask this as a DF :P We should also add a way to query the year / or quarter. This is because when going back more than 5 years, they completely changed their segment names. 
<img width="516" alt="image" src="https://user-images.githubusercontent.com/40023817/222854718-1a2de63d-9796-4295-b368-1df19cbd0517.png">